jvm/controls/src/main/kotlin/org/wycliffeassociates/otter/jvm/controls/media/SimpleAudioPlayer.kt

```diff
@@ -165,12 +165,19 @@ class SimpleAudioPlayer(
         menuItems.setAll(createPlaybackRateMenu())
     }
 
-    private fun playPauseTextBinding(): StringBinding {
+    private fun playPauseTextBinding(tooltip: Boolean = false): StringBinding {
```

Comment:
do we need it as a parameter? I thought all interactive UI elements should have tooltip